### PR TITLE
Test refinements

### PR DESCRIPTION
**Pull request by Shadowfiend**
Went a little ape here, but the commits say it all. My main goal here was to wrap up remaining compiler warnings and use receipt block info instead of `getBlock("latest")` wherever it made sense.

### CODE REVIEW

**Comment by Shadowfiend** [comment]
@liamzebedee @NicholasDotSol same, ready for review when either of you gets a chance.

File: solidity/test/TBTCSystemTest.js [hunk 1]
```diff
@@ -323,30 +323,31 @@ describe("TBTCSystem", async function() {
...
+        const {
+          receipt: {blockNumber: updateStartBlock},
+        } = truffleReceipt
+        const block = await web3.eth.getBlock(updateStartBlock)
+
+        expectEvent(truffleReceipt, "LotSizesUpdateStarted", {
+          _timestamp: new BN(block.timestamp),
+        })
+        expect(truffleReceipt.logs[0].args[0][0]).to.eq.BN(testSizes[0])
+        expect(truffleReceipt.logs[0].args[0][1]).to.eq.BN(testSizes[1])
```
Comment (liamzebedee) [inline solidity/test/TBTCSystemTest.js:336]: 👏

File: .circleci/config.yml [hunk 2]
```diff
@@ -61,12 +61,6 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - run: sudo npm install -g ganache-cli@6.4.2
```
Comment (liamzebedee) [inline .circleci/config.yml:64]: @NicholasDotSol what do we use now?

Comment (Shadowfiend) [reply .circleci/config.yml:64]: I believe something in the chain (possibly openzeppelin) just runs this in the same process.

**Pull request merged**